package/gluon-config-mode-outdoor/luasrc/lib/gluon/config-mode/wizard/0250-outdoor.lua

```diff
@@ -0,0 +1,28 @@
+return function(form, uci)
```

Comment:
I'd prefer having this as a subsection or similar of the WiFi config section with #1662.